codeqlAlerts.js

```diff
@@ -33,6 +33,8 @@ function getCodeAlerts(repos) {
         const filteredAlerts = filterCodeAlerts(alerts);
 
         filteredAlerts.forEach((alert) => {
+          if (alert.most_recent_instance.state !== "open") return;
```

Comment:
we can put this check in `filterCodeAlerts`, since that function is already filtering out on other criteria?

also FYI this will filter out both "fixed" and "dismissed" alerts which is good. https://docs.github.com/en/rest/reference/code-scanning